public/js/modal.js

```diff
@@ -0,0 +1,19 @@
+const modalSection = document.getElementById('modal-section');
+// eslint-disable-next-line no-unused-vars
+const openModal = () => {
+  modalSection.classList.add('modal-section-display-on');
+};
+
+const closeModal = () => {
+  modalSection.classList.remove('modal-section-display-on');
+  modalSection.classList.add('modal-display-off');
+};
+
+modalSection.onclick = (e) => {
+  if (e.target === modalSection) {
```

Comment:
is it necessary to check which target is?

Reply:
If i dont specify, the modal will close even if i click inside the modal.